src/docs/devices/DETA-Grid-Connect-Single-Gang-Two-Way-6951HA/index.md

```diff
@@ -56,15 +46,16 @@ Manual Flashing has not been tested on this specific model, but other models wit
 
 ## GPIO Pinouts
 
-### BK72XX-Based Models
+### Series 1 (WB3S) GPIO Pinouts
+
+_See [Pinouts on WB3S Module Datasheet](https://developer.tuya.com/en/docs/iot/wb3s-module-datasheet?id=K9dx20n6hz5n4#title-5-Interface%20pin%20definition) for more detail_
 
 | Pin    | Function                                                                          |
 | ------ | --------------------------------------------------------------------------------- |
-| P8     | Power Status of actual light, taking into account both switches  _(inverted)_     |
-| P9     | Other members have stated this is the Remote Switch status, but unable to confirm |
-| P14    | Relay,  _(includes LED)_                                                     |
-| P24    | Status LED  _(inverted)_                                                           |
-| P26    | Button  _(inverted)_
+| P24    | Status LED  _(inverted)_ |
+| P26    | Button  _(inverted)_ |
+| P14    | Relay and Button LED  |
+| P8     | Light activation status, taking into account the _local_ activation (this device) xor the _remote_ activation (another device) _(inverted)_     |
```

Comment:
This content is the same, just restructured for improved progressive disclosure.